raw_data/preprocessed/africa/gambia_english.txt

```diff
@@ -0,0 +1,221 @@
+
+Gathering analysis and management of information
+(49) [1] The Agency shall gather, analyse, manage and disseminate environmental information. [2] The Agency shall have access, subject to any other law, to all existing environmental information of a public nature and shall coordinate the management of environmental information. [3] The Agency shall determine deficiencies in existing environmental information and exchange information with other persons, including international organizations, non-governmental organizations and governmental agencies of other states. [4] The Agency shall, at such intervals as it deems fit, publish a state of the environment report which shall highlight the achievements of the Agency in the protection of the environment and the future challenges, problems, prospects and the strategies for addressing them. [5] The Agency may make other publications and adopt other measures and strategies for promoting public awareness of environmental issues and the need for conservation of natural resources.
+
+Environment in the school curriculum
+(50) The Agency may, in collaboration with the Minister responsible for education, adopt measures and strategies for the integration of environmental education in the school curriculum. 
+
+PART XI - OFFENCES
+
+General offences
+(51) A person who commits an offence under the provisions of this Act or any regulations made thereunder for which no penalty is provided, is liable on conviction: [a] in the case of a natural person, to a fine of not more than three thousand dalasis or to a term of imprisonment of not more than one year; and [b] in the case of a body corporate, to a fine of not more than twenty thousand dalasis.
+
+Offences relating to inspectors
+(52) [1] A person commits an offence if he knowingly: [a] hinders or obstructs an inspector in the execution of his duties under, this Act; [b] fails to comply with a lawful order or requirement made by an inspector in accordance with the provisions of this Act; [c] refuses an inspector entry upon any land or into any premises which he is empowered to enter by this Act; [d] impersonates an inspector; [e] refuses an inspector access to records kept in accordance with this Act; [f] fails to state or wrongly states his name or address to an inspector in the course of his duties under this Act; [g] misleads or gives wrongful information to an inspector under this Act; or [h] fails to carry out an improvement order issues by an inspector under this Act. [2] A person who commits an offence under subsection [1] is liable on conviction, [a] in the case of a natural person, to a fine of not more than four thousand dalasis or to a term of imprisonment of not more than two years; and [b] in the case of a body corporate, to a fine of not more than forty thousand dalasis.
```

Comment:
Please replace `under, this Act` with `under this Act`. Also, please replace `order issues by` with `order issued by`.